t/http_req_header.t

```diff
@@ -160,3 +160,41 @@ location /t {
 }
 --- response_body
 bar
+
+
+
+=== TEST 9: add header
+--- config
+location /t {
+    access_by_lua_block {
+        local wasm = require("resty.proxy-wasm")
+        local plugin = assert(wasm.load("plugin", "t/testdata/http_header/main.go.wasm"))
+        local ctx = assert(wasm.on_configure(plugin, 'req_hdr_add'))
+        assert(wasm.on_http_request_headers(ctx))
```

Comment:
Q: why we need to call `on_http_request_headers` function?

Reply:
I know the reason after read the doc: https://github.com/api7/wasm-nginx-module#on_http_request_headers